appveyor.yml

```diff
@@ -0,0 +1,14 @@
+version: 1.0.{build}
+image: Visual Studio 2017
+build_script:
+- cmd: >-
+    call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars64.bat"
```

Comment:
Is there a way to test with multiple MSVS versions? The old bot used 2012 for example, and I think the cmake folks used to have a 2010 bot even.

Reply:
Thank you for merging!

appveyor supports vs2013, vs2015, vs2017, but not vs2012.
https://www.appveyor.com/docs/build-environment/#build-worker-images

If you want, I will make another PR.